.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: false
```

Comment:
this flag is deprecated https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration suggest removing it.